extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/src/main/java/org/eclipse/smarthome/binding/bosesoundtouch/internal/XMLResponseHandler.java

```diff
@@ -492,6 +499,14 @@ public void characters(char[] ch, int start, int length) throws SAXException {
                 break;
             case InfoType:
                 setConfigOption(DEVICE_INFO_TYPE, new String(ch, start, length));
+                setConfigOption(Thing.PROPERTY_MODEL_ID, new String(ch, start, length));

```

Comment:
For consistency you could consider doing a static import here, so that it looks more in line :-)

PS: should you update both properties, or should we stop updating the old one?

Reply:
> PS: should you update both properties, or should we stop updating the old one?

Well - it's an open framework - you never know who is using what - so I suggest to keep both